indra/newview/llviewermenu.cpp

```diff
@@ -5249,9 +5249,19 @@ bool visible_take_object()
 	return !is_selection_buy_not_take() && enable_take();
 }
 
-bool visible_take_objects() 
+bool is_multiple_selection() 
 {
-    return visible_take_object() && (LLSelectMgr::getInstance()->getSelection()->getRootObjectCount() > 1);
+    return (LLSelectMgr::getInstance()->getSelection()->getRootObjectCount() > 1);
```

Comment:
I'm not sure this is a correct indicator of a multi selection, it is posiible to select a face on a root object and a face on child of other object. You might want to disable that if 'individual' selection or 'face' selection is on.